components/Button/index.js

```diff
@@ -0,0 +1,21 @@
+import styled from "styled-components";
+
+export const Button = ({ setProduct, produkt, productToBuy }) => {
```

Comment:
Don't use arrow functions if possible. And a typo.
```suggestion
export default function Button({ setProduct, product, productToBuy }) {
```